### ISSUE TITLE
Pre-work Checklist: Developer:Stephen Saldana

### ISSUE THREAD

**Issue by StephenTheDev1001** [hackforla/website#4328]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by StephenTheDev1001** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

- First Meeting: 2.5 hrs
- Docker Troubleshooting/Switching OS: 7hrs
- Other Env Setup: 1 hr
- Reading Onboarding Docs/First Issue: 1.5hrs

Total: 12hrs.

**Comment by HackforLABot** [comment]
## Activity Log: StephenTheDev1001
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/4328 at 03/28/2023, 08:53 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/4328 at 03/28/2023, 09:26 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/2830#issuecomment-1488143343 at 03/30/2023, 12:27 AM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/2830#issuecomment-1489839502 at 03/30/2023, 12:40 AM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/pull/4346 at 03/30/2023, 02:31 AM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/4328#issuecomment-1489999864 at 03/30/2023, 02:38 AM PDT
- StephenTheDev1001 PR was merged: https://github.com/hackforla/website/pull/4346#event-8907762974 at 04/02/2023, 07:10 PM PDT
- StephenTheDev1001 issue closed by PR 4346: https://github.com/hackforla/website/issues/2830#event-8907763110 at 04/02/2023, 07:10 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/4303 at 04/02/2023, 08:18 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/4303#issuecomment-1493589558 at 04/02/2023, 08:21 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/pull/4384 at 04/02/2023, 09:12 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/4233 at 04/03/2023, 08:19 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/4233#issuecomment-1495290708 at 04/03/2023, 08:21 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/pull/4403 at 04/03/2023, 09:01 PM PDT
- StephenTheDev1001 closed: https://github.com/hackforla/website/pull/4403#event-8919755128 at 04/03/2023, 09:05 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/pull/4404 at 04/03/2023, 10:28 PM PDT
- StephenTheDev1001 PR was merged: https://github.com/hackforla/website/pull/4384#event-8929493597 at 04/04/2023, 04:02 PM PDT
- StephenTheDev1001 issue closed by PR 4384: https://github.com/hackforla/website/issues/4303#event-8929493704 at 04/04/2023, 04:02 PM PDT
- StephenTheDev1001 PR was merged: https://github.com/hackforla/website/pull/4404#event-8929503144 at 04/04/2023, 04:04 PM PDT
- StephenTheDev1001 issue closed by PR 4404: https://github.com/hackforla/website/issues/4233#event-8929503258 at 04/04/2023, 04:04 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/4226 at 04/04/2023, 09:15 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/4226#issuecomment-1496896565 at 04/04/2023, 09:18 PM PDT
- StephenTheDev1001 unassigned: https://github.com/hackforla/website/issues/4226 at 04/06/2023, 09:25 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/4225 at 04/06/2023, 09:28 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/4225#issuecomment-1499923079 at 04/06/2023, 09:31 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/pull/4437 at 04/06/2023, 10:46 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/4437#issuecomment-1500037063 at 04/07/2023, 12:48 AM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/4226#issuecomment-1500665694 at 04/07/2023, 02:36 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/4225#issuecomment-1502646765 at 04/10/2023, 08:46 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/4473 at 04/11/2023, 07:41 PM PDT
- StephenTheDev1001 unassigned: https://github.com/hackforla/website/issues/4473 at 04/11/2023, 07:41 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/4225#issuecomment-1504474908 at 04/11/2023, 07:44 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/4487 at 04/11/2023, 08:11 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/4437#issuecomment-1504675530 at 04/11/2023, 10:35 PM PDT
- StephenTheDev1001 PR was merged: https://github.com/hackforla/website/pull/4437#event-8992063193 at 04/12/2023, 08:14 PM PDT
- StephenTheDev1001 issue closed by PR 4437: https://github.com/hackforla/website/issues/4225#event-8992063300 at 04/12/2023, 08:14 PM PDT
- StephenTheDev1001 closed as completed: https://github.com/hackforla/website/issues/4328#event-9003893647 at 04/13/2023, 11:20 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/4546#issuecomment-1514106677 at 04/18/2023, 09:13 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/4546#pullrequestreview-1391236614 at 04/18/2023, 09:18 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/4548#issuecomment-1515543666 at 04/19/2023, 05:22 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/4548#pullrequestreview-1393046281 at 04/19/2023, 05:29 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/4549#issuecomment-1515649627 at 04/19/2023, 08:09 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/4549#pullrequestreview-1393147298 at 04/19/2023, 08:14 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/4563 at 04/25/2023, 06:14 PM PDT
- StephenTheDev1001 unassigned: https://github.com/hackforla/website/issues/4563 at 04/25/2023, 06:19 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/4563 at 04/26/2023, 12:30 AM PDT
- StephenTheDev1001 unassigned: https://github.com/hackforla/website/issues/4563 at 04/26/2023, 12:34 AM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/4584#issuecomment-1528627523 at 04/28/2023, 08:49 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/4584#pullrequestreview-1406805830 at 04/28/2023, 09:03 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/4576#issuecomment-1528643358 at 04/28/2023, 09:05 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/4584#pullrequestreview-1406806340 at 04/28/2023, 09:08 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/4575#issuecomment-1528648124 at 04/28/2023, 09:30 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/4576#pullrequestreview-1406811324 at 04/28/2023, 09:38 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/4575#pullrequestreview-1406823054 at 04/28/2023, 09:47 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/4575#pullrequestreview-1415852719 at 05/06/2023, 11:40 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/4668#issuecomment-1547447065 at 05/15/2023, 01:51 AM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/3805#issuecomment-1545289300 at 05/18/2023, 10:18 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/pull/5037 at 07/19/2023, 09:35 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/5037#issuecomment-1643174749 at 07/19/2023, 09:51 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/5037#issuecomment-1649626546 at 07/25/2023, 04:08 AM PDT
- StephenTheDev1001 merged: https://github.com/hackforla/website/pull/5097#event-9930433136 at 07/26/2023, 06:55 PM PDT
- StephenTheDev1001 reopened: https://github.com/hackforla/website/pull/5097#event-9930433136 at 07/26/2023, 06:55 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5097#pullrequestreview-1548874440 at 07/26/2023, 07:01 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5105#pullrequestreview-1551193062 at 07/27/2023, 07:13 PM PDT
- StephenTheDev1001 PR was merged: https://github.com/hackforla/website/pull/5037#event-9958515313 at 07/30/2023, 07:49 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/4942#pullrequestreview-1574228469 at 08/11/2023, 11:55 AM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5160#issuecomment-1685167705 at 08/19/2023, 08:45 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5264 at 08/19/2023, 11:59 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5264#issuecomment-1685205907 at 08/20/2023, 12:05 AM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/4942#pullrequestreview-1585951204 at 08/20/2023, 12:08 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5259#pullrequestreview-1588037277 at 08/21/2023, 04:13 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5280#pullrequestreview-1588077565 at 08/21/2023, 04:59 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5275#pullrequestreview-1588081435 at 08/21/2023, 05:04 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5275#pullrequestreview-1588229348 at 08/21/2023, 08:52 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5275#pullrequestreview-1588433107 at 08/21/2023, 11:10 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/5264 at 08/22/2023, 12:40 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5289 at 08/22/2023, 09:27 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/5289 at 08/22/2023, 09:27 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5289#issuecomment-1689264226 at 08/22/2023, 09:30 PM PDT
- StephenTheDev1001 unassigned: https://github.com/hackforla/website/issues/5289 at 08/22/2023, 09:59 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5264#issuecomment-1689326429 at 08/22/2023, 10:58 PM PDT
- StephenTheDev1001 closed as completed: https://github.com/hackforla/website/issues/5264#event-10165801553 at 08/22/2023, 10:58 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5290 at 08/22/2023, 11:32 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5291 at 08/22/2023, 11:46 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5290#issuecomment-1689376773 at 08/22/2023, 11:48 PM PDT
- StephenTheDev1001 assigned: https://github.com/hackforla/website/issues/5290 at 08/22/2023, 11:48 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5290#issuecomment-1689400374 at 08/23/2023, 12:08 AM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5292 at 08/23/2023, 12:18 AM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5293 at 08/23/2023, 12:21 AM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5290#issuecomment-1689439613 at 08/23/2023, 12:34 AM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5290#issuecomment-1690868265 at 08/23/2023, 06:49 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5293#issuecomment-1690871592 at 08/23/2023, 06:54 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5293#issuecomment-1692476876 at 08/24/2023, 03:06 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5251#pullrequestreview-1594722489 at 08/24/2023, 07:21 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5300 at 08/24/2023, 09:52 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5301 at 08/24/2023, 09:59 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5302 at 08/24/2023, 10:03 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5303 at 08/24/2023, 10:06 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5304 at 08/24/2023, 10:09 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5305 at 08/24/2023, 10:11 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5306 at 08/24/2023, 10:13 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5307 at 08/24/2023, 10:24 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/5251#issuecomment-1694528818 at 08/26/2023, 05:13 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5264#issuecomment-1694817723 at 08/27/2023, 05:32 PM PDT
- StephenTheDev1001 opened: https://github.com/hackforla/website/issues/5326 at 08/27/2023, 05:41 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5290#issuecomment-1696581188 at 08/28/2023, 05:01 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5289#issuecomment-1696593903 at 08/28/2023, 05:18 PM PDT
- StephenTheDev1001 issue closed as not planned: https://github.com/hackforla/website/issues/5290#event-10217872413 at 08/29/2023, 01:15 AM PDT
- StephenTheDev1001 closed as completed: https://github.com/hackforla/website/issues/5306#event-10227070586 at 08/29/2023, 05:28 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5304#event-10227075576 at 08/29/2023, 05:29 PM PDT
- StephenTheDev1001 reopened: https://github.com/hackforla/website/issues/5306#event-10227070586 at 08/29/2023, 11:37 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5306#event-10228993302 at 08/29/2023, 11:37 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5307#issuecomment-1698583432 at 08/29/2023, 11:39 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5307#event-10229013502 at 08/29/2023, 11:39 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/issues/5305#issuecomment-1698583929 at 08/29/2023, 11:40 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5305#event-10229018149 at 08/29/2023, 11:40 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5303#event-10229024423 at 08/29/2023, 11:40 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5302#event-10229028134 at 08/29/2023, 11:41 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5301#event-10229030248 at 08/29/2023, 11:41 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5300#event-10229039778 at 08/29/2023, 11:42 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5293#event-10229042629 at 08/29/2023, 11:43 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5292#event-10229046591 at 08/29/2023, 11:43 PM PDT
- StephenTheDev1001 closed as not planned: https://github.com/hackforla/website/issues/5291#event-10229050412 at 08/29/2023, 11:43 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5401#pullrequestreview-1605954973 at 08/31/2023, 06:03 PM PDT
- StephenTheDev1001 commented on: https://github.com/hackforla/website/pull/5346#issuecomment-1702025871 at 08/31/2023, 07:28 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5346#pullrequestreview-1606126103 at 08/31/2023, 07:59 PM PDT
- StephenTheDev1001 submitted review: https://github.com/hackforla/website/pull/5346#pullrequestreview-1607479858 at 09/01/2023, 11:40 AM PDT
- StephenTheDev1001 issue closed as completed: https://github.com/hackforla/website/issues/5264#event-11065121388 at 11/26/2023, 06:17 PM PST
